extension-manifest-v3/src/store/options.js

```diff
@@ -161,15 +161,13 @@ export async function observe(property, fn) {
       value = options[property];
 
       try {
-        return await fn(value, prevValue);
+        await fn(value, prevValue);
```

Comment:
why we drop the return value from here? It may be useful is some new use-cases.

Reply:
I changed it because my linter complained. We don't use the value and the function isn't guaranteed to return a value (because of the catch).